process/best_practices_badge.md

```diff
@@ -12,7 +12,73 @@ This document outlines specific questions Academy Software Foundation projects h
 * TOC
 {:toc}
 
-## "The project MUST have at least one primary developer who knows how to design secure software." - how to fulfill this?
+## Passing
+
+### The project website MUST succinctly describe what the software does (what problem does it solve?).
+
+For projects without a website, the primary GitHub repo ```README.md``` should have this information.
+
+### The project website MUST provide information on how to: obtain, provide feedback (as bug reports or enhancements), and contribute to the software.
+
+The recommendation is to have this specifically outlined in the ```README.md``` or in a separate ```CONTRIBUTING.md``` file.
```

Comment:
Instead of "The recommendation is to have", I urge saying "We recommend having". Passive voice is notoriously hard to understand. Also, I'd note CONTRIBUTING.md first, like this:

We recommend having a separate ```CONTRIBUTING.md``` file if it's non-trivial; if it's trivial, it can be noted in the ```README.md``` file.